### ISSUE TITLE
Radio tuning does not work

### ISSUE THREAD

**Issue by Ionaru** [Stumblinbear/BetterRecords#19]
Minecraft version: 1.7.10
Better Records version: 1.2.1

When attempting to tune a frequency crystal, the GUI will say "Tuner ready!" but the "Tune" button stays disabled. Nothing was logged to the console.

![screenshot_3](https://cloud.githubusercontent.com/assets/3472373/22272535/f4773cce-e29a-11e6-95e9-5aecd90d8349.png)
Link tried: http://icecast-qmusic.cdp.triple-it.nl/Qmusic_nl_live_96.mp3

The link seems to work on Better Records version 1.2.0 (Also for Minecraft 1.7.10).

**Comment by AngrySoundTech** [comment]
Minecraft 1.7.10 is no longer being supported as we update to 1.10 and subsequently 1.11

**Comment by Ionaru** [comment]
Seems odd to me to drop support with such a huge part in the mod's functionality broken.

**Comment by AngrySoundTech** [comment]
Although I would like to continue support for 1.7.10, there are a number of reasons why this is the case.

1.  I'm not the one who wrote the 1.7.10 version. When I joined the project I did a massive refactor (see 8537719) and updated to 1.8.9, making it almost a completely different codebase

2. I'm a college student, which already limits the amount of time I have to work on this project, which I do in my free time. To maintain two concurrent branches would be difficult, and would limit the amount of time I have to work on fixing bugs in the current version and features.

3. Most if not all major mods are already updated to 1.10. With Minecraft at 1.11 now, It doesn't make sense to continue support for an obsolete version
